tests/FSharp.Data.GraphQL.IntegrationTests/FSharp.Data.GraphQL.IntegrationTests.fsproj

```diff
@@ -16,6 +16,7 @@
 
   <ItemGroup>
     <Content Include="introspection.json" />
+    <Content Include="introspection-altered-types.json" />
```

Comment:
@hth313 can we change `introspection.json` to the content of `introspection-altered-types.json` to have only one?

Reply:
@ivelten what do you think?

Reply:
@xperiandri I think we can do that, but I noticed small differences between those two. `introspection.json` does have the Human ID field as a non-nullable String type, but in the `introspection-altered-types.json` we see it as an nullable ID type. Maybe we should make both non-nullable to keep consistency for actual tests.

Reply:
I just did that and pushed it. All tests passing.